src/genie/libs/parser/ironware/tests/ShowIPOSPFNeighbor/cli/equal/documentation_output_expected.py

```diff
@@ -0,0 +1,54 @@
+expected_output = {
```

Comment:
this device supports like `vrf`? if yes, I assume this output belongs to `default` vrf.  for routing protocols, we usually have `vrf` level like below:
```
'vrf': {
    'default': {
        'neighbors': {
            '10.1.10.2': {
(snip)
```

Reply:
@tahigash Device does support VRFs however to display OSPF neighbours in a VRF the command is `show ip ospf vrf xxx neighbor`, I'm still happy to display this as the default if you wish?

Reply:
yes. please add `vrf` level to the parser. otherwise, when someone want to add `show ip ospf vrf xxx neighbor` command, it will need to have different schema. better to leverage this one.